src/Integration/Telemetry/TelemetryHelper.cs

```diff
@@ -32,8 +33,25 @@ private static string GetSonarLintVersion()
             return FileVersionInfo.GetVersionInfo(typeof(TelemetryTimer).Assembly.Location).FileVersion;
         }
 
-        public static TelemetryPayload CreatePayload(TelemetryData telemetryData, DateTimeOffset now, bool isConnected)
+        public static bool IsSonarCloud(Uri sonarqubeUri) =>
+            sonarqubeUri?.ToString().Equals("https://sonarcloud.io/", StringComparison.OrdinalIgnoreCase) ?? false;
```

Comment:
Could this be `https://www.sonarcloud.io`?

Reply:
Is there no better way than comparing the `ToString` of an URI?

Reply:
Good catch. The browser will redirect to _https://sonarcloud.io_, but if the user types in _https://www.sonarcloud.io_ then that's what we'll store in the file.